packages/pluggableWidgets/bar-chart-web/src/BarChart.editorPreview.tsx

```diff
@@ -1,27 +1,6 @@
-import { createElement, ReactNode } from "react";
-import { ChartWidget } from "@mendix/shared-charts";
+import { ReactNode } from "react";
 
 export function preview(): ReactNode {
     // TODO:
-    return (
-        <ChartWidget
-            className="widget-bar-chart"
-            data={[{ x: [], y: [], customSeriesOptions: "" }]}
-            widthUnit={"pixels"}
-            width={700}
-            heightUnit={"pixels"}
-            height={450}
-            showLegend={false}
-            xAxisLabel={undefined}
-            yAxisLabel={undefined}
-            gridLinesMode={"both"}
-            showSidebarEditor={false}
-            customLayout={""}
-            customConfig={""}
-            configOptions={{}}
-            layoutOptions={{}}
-            seriesOptions={{}}
-            type="BarChart"
-        />
-    );
+    return null;
```

Comment:
Just curious why you did this to all editorPreview files? It's because it doesn't render in design mode and you need svg?

Reply:
I till be replaced by an SVG. Currently its not a good solution to bundle the whole plotly inside the editorPreview file. (It also kills CI)